flink-ml-lib/src/main/java/org/apache/flink/ml/evaluation/binaryclassification/BinaryClassificationEvaluator.java

```diff
@@ -668,30 +669,64 @@ public Tuple3<Double, Boolean, Double> map(Row value) throws Exception {
     }
 
     /** Binary Summary of data in one worker. */
-    private static class BinarySummary implements Serializable {
-        private final Integer taskId;
+    public static class BinarySummary implements Serializable {
+        private Integer taskId;
         // maximum score in this partition
         private double maxScore;
         // real positives in this partition
         private long curPositive;
         // real negatives in this partition
         private long curNegative;
 
+        public BinarySummary() {}
+
         public BinarySummary(Integer taskId, double maxScore, long curPositive, long curNegative) {
             this.taskId = taskId;
             this.maxScore = maxScore;
             this.curPositive = curPositive;
             this.curNegative = curNegative;
         }
+
+        public Integer getTaskId() {
```

Comment:
Would it be simpler to make these variables `public` than adding get/set methods?